sematic/ui/packages/common/src/pages/RunSearch/filters/OtherFilterSection.tsx

```diff
@@ -25,18 +27,36 @@ const StyledFormControlLabel = styled(FormControlLabel)`
     margin-left: ${theme.spacing(2.9)};
 `;
 
-interface OwnersFilterSectionProps {
+interface OtherFilterSectionProps {
     onFiltersChanged?: (filters: string[]) => void;
 }
 
-const OtherFiltersSection = (props: OwnersFilterSectionProps) => {
+const OtherFiltersSection = forwardRef<ResettableHandle, OtherFilterSectionProps>((props, ref) => {
+    const { onFiltersChanged } = props;
+    const [rootRunsOnly, setRootRunsOnly] = useState<boolean>(false);
+
+    useImperativeHandle(ref, () => ({
+        reset: () => {
+            setRootRunsOnly(false);
+        }
+    }));
+
+    const onRootRunsOnlyChanged = useCallback((checked: boolean) => {
+        setRootRunsOnly(checked);
+        onFiltersChanged?.(checked ? ["rootRunsOnly"] : []);
+    }, [setRootRunsOnly, onFiltersChanged]);
+
+
     return <StyledCollapseableFilterSection title={"Filters"} >
         <Container>
             <FormGroup>
-                <StyledFormControlLabel control={<Checkbox defaultChecked />} label="Root runs only" />
+                <StyledFormControlLabel control={
+                    <Checkbox checked={rootRunsOnly} onChange={(_, checked) => {
+                        onRootRunsOnlyChanged(checked);
```

Comment:
Update `onRootRunsOnlyChanged` to take the `ChangeEvent` value to improve readability?